backend/common/repo.py

```diff
@@ -306,31 +327,40 @@ def _get_release_index_from_file(self) -> typing.Dict[str, "DpkgRepo.ReleaseEntr
             elif os.access(self._get_parent_url(local_path, 0, "Release"), os.R_OK):
                 release_file = self._get_parent_url(local_path, 0, "Release")
             else:
+                logging.exception("No release file found in '{}'. Raising \
```

Comment:
No, this is `logging.exception`, since it is also printing the stack trace.

Reply:
I came across this because the `logging.exception` documentation says not to use it outside of exception handling.

I don't understand why we need to log the stack trace here and not where we handle the exception.

Reply:
Forget it. Must have been a copy&paste error and I didn't notice. Sorry!